lib/voyager/clients/microsoft_graph_client.rb

```diff
@@ -51,22 +51,38 @@ def my_drives
       get('/me/drives')
     end
 
-    def my_drive
-      get('/me/drive')
+    def my_drive_children
+      get('/me/drive/root/children')
     end
 
     def my_drive_items(item_id = 'root', options = '')
       get("/me/drive/#{item_id}/children" + options)
     end
 
-    def drive(drive_id, query_opts = '')
+    def following
```

Comment:
I would name this slightly more descriptively (e.g.` def drives_following`) since sharepoint is confusing, so any extra descriptiveness in the code is helpful and all other drive methods are prefixed with dirves